From acaf08d1bbbfe6ad45d184becd17630fb077a447 Mon Sep 17 00:00:00 2001
From: Laura Abbott <labbott@redhat.com>
Date: Wed, 19 Jun 2019 16:28:24 -0400
Subject: [PATCH 35/35] redhat: configs: Set configuration values for
 secureboot

Message-id: <20190619162824.427-20-labbott@redhat.com>
Patchwork-id: 265697
O-Subject: [ARK PATCH 35/35] redhat: configs: Set configuration values for secureboot
Bugzilla:
RH-Acked-by: Jarod Wilson <jarod@redhat.com>

We need initial values to go with the secure boot configuration options.
Keep them all off temporarily until we verify what we want to test.

Signed-off-by: Laura Abbott <labbott@redhat.com>
---
 redhat/configs/ark/generic/CONFIG_KEXEC_SIG                    | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL             | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE       | 1 +
 4 files changed, 4 insertions(+)
 create mode 100644 redhat/configs/ark/generic/CONFIG_KEXEC_SIG
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE

Signed-off-by: Laura Abbott <labbott@redhat.com>
---
 redhat/configs/ark/generic/CONFIG_KEXEC_SIG                    | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL             | 1 +
 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE       | 1 +
 4 files changed, 4 insertions(+)
 create mode 100644 redhat/configs/ark/generic/CONFIG_KEXEC_SIG
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL
 create mode 100644 redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE

diff --git a/redhat/configs/ark/generic/CONFIG_KEXEC_SIG b/redhat/configs/ark/generic/CONFIG_KEXEC_SIG
new file mode 100644
index 000000000000..49392e48598b
--- /dev/null
+++ b/redhat/configs/ark/generic/CONFIG_KEXEC_SIG
@@ -0,0 +1 @@
+# CONFIG_KEXEC_SIG is not set
diff --git a/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT
new file mode 100644
index 000000000000..336528547ec3
--- /dev/null
+++ b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT
@@ -0,0 +1 @@
+# CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT is not set
diff --git a/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL
new file mode 100644
index 000000000000..c22c35ff2a67
--- /dev/null
+++ b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL
@@ -0,0 +1 @@
+# CONFIG_LOCK_DOWN_KERNEL is not set
diff --git a/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE
new file mode 100644
index 000000000000..b6a333ecd88a
--- /dev/null
+++ b/redhat/configs/ark/generic/CONFIG_LOCK_DOWN_KERNEL_FORCE
@@ -0,0 +1 @@
+# CONFIG_LOCK_DOWN_KERNEL_FORCE is not set
-- 
2.21.0

